addons/base_vat/models/res_partner.py

```diff
@@ -112,15 +113,18 @@ def simple_vat_check(self, country_code, vat_number):
     def _check_vies(self, vat):
         # Store the VIES result in the cache. In case an exception is raised during the request
         # (e.g. service unavailable), the fallback on simple_vat_check is not kept in cache.
-        return check_vies(vat)
+        try:
+            return check_vies(vat)
+        except Fault:
+            return False
 
     @api.model
     def vies_vat_check(self, country_code, vat_number):
         try:
             # Validate against  VAT Information Exchange System (VIES)
             # see also http://ec.europa.eu/taxation_customs/vies/
             vies_result = self._check_vies(country_code.upper() + vat_number)
-            return vies_result['valid']
+            return vies_result and vies_result['valid']
         except InvalidComponent:
             return False
         except Exception:
```

Comment:
What you might want to do is set `sentry_ignored` here for the `_logger.exception`
Not sure if it works @yelizariev ?